README.md

```diff
@@ -159,6 +173,74 @@ ember exam --split=3 --partition=3
 
 **Note 2**: _You must be using Testem version `1.5.0` or greater for this feature to work properly._
 
+### Test Load Balancing
+
+```bash
+$ ember exam --parallel=<num> --load-balance
+```
+
+`load-balance` allows you to load balance test files against. It will order the test files by test type and load balance test files between the browsers dynamically rather than statically. `parallel` must be used along with `load-balance` to specify a number of browser(s).
+
+`load-balance` can not be used without `parallel` being specified a number of browser(s).
+
+This option was added to version 1.1 to address execution performance when running against a large test suite.
+
+For example, if you specify load-balance and parallel equals 3, then three browser instances will be created and the output will look something like::
+
+```bash
+ok 1 Chrome 66.0 - Browser Id 1 - some test
+ok 2 Chrome 66.0 - Browser Id 2 - some another test
+ok 3 Chrome 66.0 - Browser Id 3 - some the other test
+```
+
+You can also specify the `split` and `partition` options with `load-balance` to load a portion of test modules on multiple CI containers.
+
+```bash
+$ ember exam --split=<num> --partition=<num> --parallel=<num> --load-balance
+```
+
+This command will split test files and load-balance tests from the specified partition across the browsers. For example --split=2 -partition=1 --parallel=3 --load-balance, the complete list of test files are split into two halves. With the first half of the list load balanced against three browsers. The output will look something like below:
+```base
+ok 1 Chrome 66.0 - Exam Partition 1 - browser Id 1 - some test
+ok 2 Chrome 66.0 - Exam Partition 1 - browser Id 2 - another test
+ok 3 Chrome 66.0 - Exam Partition 1 - browser Id 3 - some the other test
+```
+
+Due to the dynamic nature of the `load-balance` option, test file execution order could vary between runs. To reproduce a past test execution follow the next section.
+
+**Note 1** _You must be using ember-cli version 3.2.0 or greater for this feature to work properly._
+**Note 2** _You must be using ember-qunit version 4.1.1 or greater for this feature to work properly._
+**Note 3** _You must be using qunit version 2.8.0 or greater for this feature to work properly._
+**Note 4** _This feature is not currently supported by Mocha._
+
+##### Test Failure Reproduction
+
+Since Ember Exam with `load-balance` loads modules one at a time the order of executing modules is not deterministic. For this reason a test execution with `load-balance` creates a json file to track which browser executes which modules in which order and the file is stored in the root directory and the naming structure is `test-execution-YYYY-MM-DD--HH-MM-SS.json`.
```

Comment:
do we need to document usage of `write-execution-file` ?

Reply:
added